### PR TITLE
400ZR FEC Uncorrectable Frames Streaming test.

### PR DESCRIPTION
**Pull request by ahsaanyousaf**

### CODE REVIEW

File: feature/platform/transceiver/zr_fec_uncorrectable_frames_test/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# TRANSCEIVER-9: Telemetry: 400ZR Optics FEC(Forward Error Correction) Uncorrectable Frames Streaming.
+
+## Summary
+
+Validate 400ZR optics module reports uncorrectable FEC frames count.
+
+This observable represents the number of uncorrectable FEC frames,
+measured as RS(544,514) equivalent frames, in a short interval.
+This is a post-FEC decoder errormetric.
```
Comment (ejbrever) [inline feature/platform/transceiver/zr_fec_uncorrectable_frames_test/README.md:9]: nit, missing space between error/metric

File: feature/platform/transceiver/zr_fec_uncorrectable_frames_test/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        example 193.1 THz
+*   With the ZR link established as explained above, verify that the
+    following ZR transceiver telemetry path exist and is streamed for both
+    the ZR optics.
+    *   /terminal-device/logical-channels/channel/otn/state/fec-uncorrectable-blocks
+*   Verify that the reported data should be of type yang:counter64.
+*   When the modules or the devices are still in a boot stage, they must not
+    stream any invalid string values like "nil" or "-inf".
+*   Toggle the interface state using /interfaces/interface/config/enabled and
+    verify relevant fec uncorrectable frame count is streamed properly.
```
Comment (ejbrever) [inline feature/platform/transceiver/zr_fec_uncorrectable_frames_test/README.md:29]: what is streamed properly in this case? Do we expect the counter to start back at zero?

Comment (ejbrever) [inline feature/platform/transceiver/zr_fec_uncorrectable_frames_test/README.md:29]: nit, FEC should be capitalized.

File: feature/platform/transceiver/zr_fec_uncorrectable_frames_test/README.md (hunk 1 above)
Comment (ahsaanyousaf) [reply feature/platform/transceiver/zr_fec_uncorrectable_frames_test/README.md:9]: Done

File: feature/platform/transceiver/zr_fec_uncorrectable_frames_test/README.md (hunk 2 above)
Comment (ahsaanyousaf) [reply feature/platform/transceiver/zr_fec_uncorrectable_frames_test/README.md:29]: Done

**Review by ejbrever — APPROVED** [review]

**Review by dplore — CHANGES_REQUESTED** [review]
Please add testregistry.textproto entry

**Review by dplore — APPROVED** [review]

**Pull request merged**